apps/site/assets/ts/tnm/components/TransitNearMe.tsx

```diff
@@ -42,33 +45,25 @@ const validateModeFilter = (acc: Mode[], mode: string): Mode[] =>
 export const modesFromQuery = (query: QueryParams): Mode[] =>
   query.filter ? query.filter.split(",").reduce(validateModeFilter, []) : [];
 
-export const fetchData = (
-  query: QueryParams,
+export const fetchRealtimeSchedules = async (
+  stops: Stop[],
   dispatch: Dispatch
-): Promise<void> => {
-  if (
-    (query.latitude || query["location[latitude]"]) &&
-    (query.longitude || query["location[longitude]"]) &&
-    window.fetch
-  ) {
-    return window
-      .fetch(
-        `/transit-near-me/api${paramsToString(
-          query,
-          window.encodeURIComponent
-        )}`
-      )
-      .then((response: Response) => {
-        if (response.ok) return response.json();
-        throw new Error(response.statusText);
-      })
-      .then((routes: RouteWithStopsWithDirections[]) =>
-        dispatch(routeSidebarDataAction(routes))
-      )
-      .catch(() => {});
-  }
+): Promise<undefined> => {
+  if (!window.fetch) return;
```

Comment:
More appropriate to make the SSR comment here: we trade off SSR rendering of the route cards for a faster initial page load without the cards.

Reply:
at an earlier stage we determined that it currently isn't possible for us to build all the data to render all the cards fast enough in a single request.  The main feature we are getting with this change is the ability to do the data fetching in smaller chunks.  However, in a future optimization we might send down the data for the first few cards so that they are rendered ~instantly, but that is out of scope for first pass

Reply:
I think once this new foundation is in place there are a few potential next steps we could take to improve the user experience... but I'm hesitant to put more into this PR as it's already take so long :/